doc/world_format.txt

```diff
@@ -281,6 +282,8 @@ MapBlock serialization format
 NOTE: Byte order is MSB first (big-endian).
 NOTE: Zlib data is in such a format that Python's zlib at least can
       directly decompress.
+NOTE: In version 29 zstd is used instead of zlib. In addition the entire
+      block is first serialized and then compressed.
```

Comment:
```suggestion
NOTE: Since version 29 zstd is used instead of zlib. In addition the entire
      block is first serialized and then compressed (except the version byte).
```